### PR TITLE
Refactoring of RSE service pages

### PR DESCRIPTION
**Pull request by willfurnass**
* Add links to Projects page and testimonials page
* s/group/team
* Add another link to contacts page
* Note that line management remains with RSE team for RSEs deployed on projects
* Minor tweaks to info on section on directly allocated staff vs day rate service

### CODE REVIEW

File: pages/service/index.md [hunk 1]
```diff
@@ -57,11 +59,11 @@ the RSE service is a paid service.
 In many cases this will require you to budget for the RSE service as part of your grant application.
 The RSE team can assist with this an advice you as to what is appropriate for your needs.
 Enquiries should be made to `rse@sheffield.ac.uk`.
-The RSE group allows you to budget for RSE services using a day rate charge.
+The RSE team allows you to budget for RSE services using a day rate charge.
```
Comment (willfurnass) [inline pages/service/index.md]: @mondus: any objections to moving *The RSE team allows you to budget for RSE services using a day rate charge* to the next section on the *RSE service charge*?

File: pages/service/index.md [hunk 2]
```diff
@@ -57,11 +59,12 @@ the RSE service is a paid service.
 In many cases this will require you to budget for the RSE service as part of your grant application.
 The RSE team can assist with this an advice you as to what is appropriate for your needs.
 Enquiries should be made to `rse@sheffield.ac.uk`.
-The RSE group allows you to budget for RSE services using a day rate charge.
-We can provide staffing for anything from 1 day to 50% of a FTE.
+
+We can provide directly allocated staffing for your grants for anything from 1 day to 50% of a FTE.
```
Comment (bobturneruk) [inline pages/service/index.md]: I can only assume this is the correct terminology!

File: pages/service/provision/provision.md [hunk 3]
```diff
@@ -17,7 +17,8 @@ Please see the flow chart below for our service provision model.
 
 ## RSE line management
 
-Once an RSE has been allocated to you then you will provide day-to-day line management of the staff member for the allocated time of the project.
+Once an RSE has been allocated to you then you will direct the activities of the RSE within the scope of the project but line management of the RSE will remain within the RSE team.
```
Comment (bobturneruk) [inline pages/service/provision/provision.md]: How about?

> Once an RSE has been allocated they will engage and collaborate with you and your team directly. Line management of the RSE will remain within the RSE team.

**Review by bobturneruk — APPROVED** [review]
Great! Important changes. I'd love to see this merged swiftly.

File: pages/service/index.md (hunk 2 above)
Comment (willfurnass) [reply pages/service/index.md]: Hmm.  @mondus this should actually be 'directly incurred' instead, right?  Think we actually want something like:

 > We can provide staffing for your grants (charged as directly incurred costs) for anything from 1 day to 50% of a FTE.

Comment (willfurnass) [reply pages/service/index.md]: Think this has been addressed.

**Review by mondus — APPROVED** [review]
Happy with this.

**Comment by willfurnass** [comment]
Ta

**Pull request merged**